app/views/foods/_food.html.erb

```diff
@@ -2,8 +2,10 @@
   <th scope="row"><%= food.name%></th>
   <td><%= food.measurement_unit %></td>
   <td>$ <%= food.price %></td>
-  <td>
-    <%= button_to "Delete Food" ,"foods/#{food.id}", method: :delete, data:
-    {confirm: "Are you sure"}, :class => "btn btn-sm btn-danger"%>
-  </td>
+  <% if can? :destroy, food %>
+    <td>
+      <%= button_to "Delete Food" ,"foods/#{food.id}", method: :delete, data:
+      {confirm: "Are you sure"}, :class => "btn btn-sm btn-danger"%>
+    </td>
+  <% end %>
```

Comment:
Kindly put the `if can` statement inside the `<td>` tag to avoid breaking the table's style

Reply:
Done!